### ISSUE TITLE
Set estimated time

### ISSUE THREAD

**Issue by Inrego** [go-vikunja/api#46]
I would like to be able to estimate how long time a task will take to finish. There are many use cases for this, but off the top of my head (for private use), I'd like to sort by estimated time, so if I have just an hour to do something, I can quickly find a task on my list to finish.

**Comment by salco** [comment]
Yes, is there any issue to implement this kind of feature. I think it could help plan task load.

**Comment by bitzerk** [comment]
This is the one feature missing that's holding me back from using Vikunja. I need the ability to store my estimate on tasks - it helps me decide where to place that task on the calendar. Bonus points if the estimate could show on each task bubble in the gantt chart view (which is awesome, btw).

**Comment by alexbonhomme** [comment]
Yes I need this feature too! I was very surprise to do not find it right away.
Thanks for your work!

**Comment by kergma-lw** [comment]
@kolaente I'm considering working on this. I checked the current task model and task details UI: time tracking is available, but I couldn't find a task effort estimate or sorting/filtering by it.

Would you be open to a contribution for this feature? Is there any existing work or planned direction I should coordinate with before putting together a proposal?
